src/components/common/SwapPrice.stories.tsx

```diff
@@ -19,6 +19,7 @@ const Template: Story<Partial<Props>> = (props) => {
     <SwapPrice
       baseToken={GNO}
       quoteToken={DAI}
```

Comment:
Same as with `isPriceInverted`, but worse
Either use Controls like they are meant to be used, optionally with selector like I do in [Orderbook stories](https://github.com/gnosis/dex-react/blob/8913bd51f41962cad94376506b2769d15855d13a/src/components/OrderBookWidget.stories.tsx#L34) or disable token controls altogether.

Otherwise you can break them in storybook

![pr1424--dexreact review gnosisdev com_storybook__path=_story_common-swapprice--basic](https://user-images.githubusercontent.com/5121491/93569056-b7f7bc00-f999-11ea-9f87-280091129e9b.png)

Reply:
I'm doing this in a new PR, since it's unrelated to this one https://github.com/gnosis/dex-react/pull/1425